tests/odeint_tests.py

```diff
@@ -65,6 +65,13 @@ def test_adaptive_heun(self):
             with self.subTest(ode=ode):
                 self.assertLess(rel_error(sol, y), error_tol)
 
+    def test_dopri8(self):
+        for ode in problems.PROBLEMS.keys():
+            f, y0, t_points, sol = problems.construct_problem(TEST_DEVICE, ode=ode)
+            y = torchdiffeq.odeint(f, y0, t_points, method='dopri8')
```

Comment:
This test isn't passing for me. Can you set a lower tolerance (atol=1e-12, rtol=1e-12) so this test passes?

Reply:
I've set the tolerance as rtol=1e-12, atol=1e-14. All tests passed successfully on my machine